avocado/core/nrunner.py

```diff
@@ -604,11 +604,9 @@ def _get_env_variables(self):
         return avocado_test_env_variables
```

Comment:
I can't find the reference for commit 2e1c905c, so I'm losing some context here.

Reply:
yes it is commit left over from previous version, it should be 080653d4871783ec051e38477210997440b9e5af. thanks I will update that.